QSB/MeteorSync/MeteorManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using QSB.MeteorSync.WorldObjects;
+using QSB.WorldSync;
+
+namespace QSB.MeteorSync
+{
+	public class MeteorManager : WorldObjectManager
+	{
+		public static bool MeteorsReady;
+
+		protected override void RebuildWorldObjects(OWScene scene)
+		{
+			QSBWorldSync.Init<QSBMeteorLauncher, MeteorLauncher>();
+			// wait a bit because meteors get created late
+			QSBCore.UnityEvents.FireInNUpdates(() =>
+			{
+				QSBWorldSync.Init<QSBMeteor, MeteorController>();
+				MeteorsReady = true;
+			}, 10);
```

Comment:
you can put code *blocks* in these?? 😵 

that would have saved a lot of code previously :P